FreeRTOS-Plus/Demo/coreMQTT_Windows_Simulator/MQTT_Multitask/DemoTasks/MultitaskMQTTExample.c

```diff
@@ -2218,7 +2106,7 @@ static void prvMQTTDemoTask( void * pvParameters )
         xNetworkStatus = prvSocketDisconnect( &xNetworkContext );
         configASSERT( xNetworkStatus == pdPASS );
 
-        LogInfo( ( "prvMQTTDemoTask() completed an iteration successfully. Total free heap is %u.\r\n", xPortGetFreeHeapSize() ) );
+        LogInfo( ( "\r\n\r\nprvMQTTDemoTask() completed an iteration successfully. Total free heap is %u.\r\n", xPortGetFreeHeapSize() ) );
```

Comment:
If the intention of `prvCleanExistingPersistentSession();` is to also clean up sessions from the current execution, should we call it right before this line?

Reply:
The code should behave as it did before, I've have just removed the code that cleaned up the prior session into its own function.